persistence/src/commonMain/kotlin/com/wire/kalium/persistence/dao/UserDAO.kt

```diff
@@ -11,17 +11,22 @@ data class QualifiedID(
 
 typealias UserId = QualifiedID
 
-data class User(
+data class UserEntity(
     val id: QualifiedID,
     val name: String?,
-    val handle: String?
+    val handle: String?,
+    val email: String?,
+    val phone: String?,
+    val accentId: Int,
+    val team: String?
+    // val picture: List<UserAsset>
```

Comment:
Since we only have two types of assets II think we can unroll the list of UserAssets into:
`preview_asset_id: String?`
`complete_asset_id: String?`

Reply:
I added a TODO comment, that will be addressed on upcoming PR, when for example we have an `assets` table to map this field with 👍

Reply:
OK 👍 

My suggestion was that we don't necessarily need an `assets` table and we could include the asset data directly in the user table. But we can discuss further in the upcoming PR then.